person.rb

```diff
@@ -21,6 +20,10 @@ def correct_name
     @name
   end
 
+  def add_rental(rental)
+    @rentals << rental
+  end
+
```

Comment:
- [ ] Firstly, kindly add  getters and setters for `rentals`  as a `person` should have many rentals 
- [ ] Secondly, `@rentals`  should be initialized  in your constructor Link to the example about constructors in a previous [lesson]( https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/class_and_object_for_irb.md#constructor-and-instance-vars)
- [ ] Thirdly, The `person` itself should be sent as a parameter to create the new rental achieving the 'has-many' association. For that, add a `date` to the parameter of the method. This should refactor your method to somthing like this:
### Sample
```
Rental.new(date, book, self)
end
```

Reply:
Needs attention!